src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+
+    @Override
+    public List<Truck> get() {
+        return List.of(new Truck(), new Truck());
+    }
```

Comment:
You didn't create a few models of Trucks. But in Bulldozer and Excavator you did.

Reply:
In fact I created two separate objects of the truck type just without any parameters

Reply:
Yes, but in other producers you created Machines with parameters. So it may look unlogical for a mentor.